release-please-config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "bootstrap-sha": "13183350fac680be8c0e89ca3dc3951330b8b7c0",
```

Comment:
I intentionally defined properties with default valued because they might be used by someone later.

Reply:
btw it is a hash of a commit (not including) from which it will make next release